envs/cytominer_env.yaml

```diff
@@ -0,0 +1,10 @@
+name: ctyominer_env
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - python=3.7.0
+  - pip
+  - pip:
+    - "git+https://github.com/cytomining/pycytominer@1279cd2349715604d226b58e7aa61a8a8eda7e54"
```

Comment:
pycytominer is now on pip! https://pycytominer.readthedocs.io/en/latest/install.html (please use appropriate version, IIRC `0.1`)

Reply:
Cool! I've implemented the current version!

d74575671a20747cfe0ac21744b162584cc915de